.gitignore

```diff
@@ -1,9 +1,10 @@
 # picolibrary-microchip-megaavr
 #
-# Copyright 2020, 2021 Andrew Countryman <apcountryman@gmail.com>
+# Copyright 2020-2021, Andrew Countryman <apcountryman@gmail.com> and the
+# picolibrary-microchip-megaavr contributors
 #
-# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
-# file except in compliance with the License. You may obtain a copy of the License at
+# Copyright 2020-2021, Andrew Countryman <apcountryman@gmail.com> and the
+# picolibrary-microchip-megaavr contributors
```

Comment:
These two lines are from the Apache License text and thus should not be modified.